raiden_contracts/tests/test_print_gas.py

```diff
@@ -156,7 +155,7 @@ def print_gas_channel_cycle(
 ) -> None:
     """Abusing pytest to print gas costs of TokenNetwork's operations"""
     (A, B, C, D) = get_accounts(4)
-    settle_timeout = 11
+    settle_timeout = TEST_SETTLE_TIMEOUT
```

Comment:
Before the `settle_timeout` was always set to `8` and we replaced it with `TEST_SETTLE_TIMEOUT` which was the equivalent. But here it is now explicitly `11`. Either this was unintended, but to me this reads like something special. Why else should the developer do this here. So do we need to take this into account here? Because now we just ignore it.

Reply:
Setting the settle timeout is no longer a valid thing. We have this value hardcoded, therefore this value is no longer valid. We're still using this variable bellow to check the final result of the settlement.